coldfront/core/utils/management/commands/audit_data.py

```diff
@@ -0,0 +1,323 @@
+'''
```

Comment:
FYI: Prior to launching MyLRC, I wrote `core.utils.management.commands.validate_lrc_initial_state`. It doesn't really cover the LRC checks you'll need to do here, but you can use it as a reference.